File: Modul 2/C/PRAK205-2310817310010-AkmallullailSyaban.c
#include <stdio.h>
#include <math.h>

int main(){
    float a, b;
    
    printf("Input\n");
    scanf("%f %f", &a, &b);
    
    float c = sqrt(b*b - a*a);
    float tinggi = a;
    float alas = c;
    float keliling = a + b + c;
    float luas = (alas * tinggi) / 2;

    printf("\nOutput\n");
    printf("Alas     = %.0f cm\n", alas);
    printf("Tinggi   = %.0f cm\n", tinggi);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas     = %.0f cm^2\n", luas);

    return 0;
}

File: Modul 2/python/PRAK205-2310817310010-AkmallullailSyaban.py
import math

angka = input().split()
a, b = map(int,angka)
c = int(math.sqrt( b ** 2 - a ** 2))
keliling = int(a + b + c)
luas = int(c * a) / 2

print(f"Alas     = {c:.0f} cm")
print(f"Tinggi   = {a:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas     = {luas:.0f} cm^2")